packages/ui/src/components/GalleryPost/GalleryPost.tsx

```diff
@@ -5,28 +5,40 @@ import {
   isReferencePost,
   isTextPost,
   textPostIsLinkedImage,
+  tiptap,
 } from '@tloncorp/shared/dist';
 import * as db from '@tloncorp/shared/dist/db';
 import { Link } from 'packages/shared/dist/urbit';
-import { useMemo } from 'react';
-import { Dimensions, ImageBackground } from 'react-native';
+import { useCallback, useMemo } from 'react';
+import {
+  Dimensions,
+  Image,
+  ImageBackground,
+  TouchableOpacity,
+} from 'react-native';
 
-import { LinearGradient, View } from '../../core';
+import { LinearGradient, Text, View, YStack } from '../../core';
 import ChatContent from '../ChatMessage/ChatContent';
 import ContentReference from '../ContentReference';
 import Pressable from '../Pressable';
 import GalleryAuthorRow from './GalleryAuthorRow';
 
 const HEIGHT_AND_WIDTH = (Dimensions.get('window').width - 46) / 2;
+const HEIGHT_DETAIL_VIEW = Dimensions.get('window').height - 236;
```

Comment:
Would be nice to have some comments here to explain what's happening w/ the magic numbers.